ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java

```diff
@@ -78,12 +80,13 @@ public ContainerBuilder(Container origin) {
    * @param description The description of the {@link Container}.
    * @param parentAccountId The id of the parent {@link Account} of the {@link Container} to build.
    */
-  public ContainerBuilder(short id, String name, ContainerStatus status, String description, short parentAccountId) {
+  public ContainerBuilder(short id, String name, ContainerStatus status, String description, short parentAccountId, long lastUpdateTime) {
```

Comment:
The beauty of having builder pattern is to avoid this massive change when adding a new field. I suggest that we make lastUpdateTime as an optional here. We can give lastUpdateTime a default value and only set the lastUpdateTime when it's not the default value.

```
public ContainerBuilder(short id, String name, ContainerStatus status, String description, short parentAccountId) {
    this.id = id;
    this.name = name;
    this.status = status;
    this.description = description;
    this.parentAccountId = parentAccountId;
}

public ContainerBuilder lastUpdateTime(long val) {
    this.lastUpdateTime = val;
    return this;
}
```

Reply:
Updated to make lastUpdateTime optional.